pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -259,7 +259,7 @@ void processArguments() throws Exception {
         protected Boolean retainOrdering;
         @Parameter(names = "--subs-name", description = "Pulsar source subscription name if user wants a specific subscription-name for input-topic consumer")
         protected String subsName;
-        @Parameter(names = "--parallelism", description = "The function's parallelism factor (i.e. the number of function instances to run)")
+        @Parameter(names = "--parallelism", description = "The parallelism factor of function (i.e. the number of function instances to run)")
```

Comment:
"The parallelism factor of a Pulsar Function (i.e. the number of function instances to run)"